tests/README.md

```diff
@@ -37,3 +37,49 @@ MOONBEAM_LOG="warn,rpc=trace" npm run test
 
 The test script will find available ports above 20000 in order to ensure that it doesn't conflict
 with any other running services.
+
+# Debugging a Moonbeam node
+
+The repository contains a pre-configured debugger configuration for VSCode with the **CodeLLDB**
+(`vadimcn.vscode-lldb`) extension. In the Debug tab of VSCode the dropdown at the top list 2
+configurations :
+
+- **Build & Launch Moonbeam Node (Linux)** : cargo build the node with debug profile and launch it.
```

Comment:
please write down the exact command for build and the exact command for launch. I think it's the same as macOS

Reply:
The profile did it for you. It has now been removed to better include debug symbols, and to build you just have to run `./scripts/debugger-build.sh`

Reply:
Added the command to the README.md and also as a VSCode task.